exec/java-exec/src/test/java/org/apache/drill/exec/vector/complex/writer/TestPromotableWriter.java

```diff
@@ -33,6 +33,7 @@
   public void list() throws Exception {
     BufferAllocator allocator = RootAllocatorFactory.newRoot(DrillConfig.create());
     TestOutputMutator output = new TestOutputMutator(allocator);
+    @SuppressWarnings("resource")
```

Comment:
Why is it necessary to suppress resource not being closed warning here and OK to remove warning suppression in other places?